File: Jungle_Camping/Jungle_Camping.java
import java.util.Scanner;

public class Program
{
  public static void main(String[] args) 
{
    Scanner m = new Scanner(System.in);
    String n = m.nextLine();
    String[] t = n.split(" ");
    for(var x : t)
    {
    System.out.print(sta(x)+" ");
    }
}
public static String sta(String n)
{
    if(n.equals("Grr"))
    {
      return "Lion";
    }
    if(n.equals("Rawr"))
    {
      return "Tiger";
    }
    if(n.equals("Ssss"))
    {
      return "Snake";
    }
    if(n.equals("Chirp"))
    {
      return "Bird";
    }
    return "";
}
}

File: Jungle_Camping/Jungle_Camping.cpp
#include <iostream>
#include <string>
using namespace std;
string sta(string& n);
int main()
{
    string in,out;
    while(cin>>in)
    {
     out += sta(in)+" ";
    }
    cout<<out;
}

string sta(string & n)
{
    if(n == "Grr")
    {
      return "Lion";
    }
    if(n == "Rawr")
    {
      return "Tiger";
    }
    if(n == "Ssss")
    {
      return "Snake";
    }
    if(n == "Chirp")
    {
      return "Bird";
    }
    return "";
}
